### PR TITLE
docs: aggregate targets

### PR DESCRIPTION
**Pull request by knqyf263**
## Description
This PR puts all the targets in a single place for clarity. Each target may have specific options.

### Before
image

### After
image

## Related PRs
- [x] https://github.com/aquasecurity/trivy/pull/3495
- [x] https://github.com/aquasecurity/trivy/pull/3496

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: docs/docs/target/filesystem.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+### Secrets
+It is enabled by default.
+See [here](../secret/scanning.md) for the detail.
+
+```shell
+$ trivy fs /path/to/project
+```
+
+### Secrets
```
Comment (DmitriyLewen) [inline docs/docs/target/filesystem.md:83]:
```suggestion
### Licenses
```

File: docs/docs/target/git-repository.md [hunk 2]
```diff
@@ -147,32 +164,60 @@ Total: 20 (UNKNOWN: 3, LOW: 0, MEDIUM: 7, HIGH: 5, CRITICAL: 5)
...
+
+### Secrets
+It is enabled by default.
+See [here](../secret/scanning.md) for the detail.
+
+```shell
+$ trivy repo [YOUR_REPO_URL]
+```
+
+### Secrets
```
Comment (DmitriyLewen) [inline docs/docs/target/git-repository.md:183]:
```suggestion
### Licenses
```

Comment (knqyf263) [reply docs/docs/target/git-repository.md]: Fixed in https://github.com/aquasecurity/trivy/pull/3503/commits/adc9fce4459cc882f49d138b56c5d6a805e45f31

File: docs/docs/target/filesystem.md (hunk 1 above)
Comment (knqyf263) [reply docs/docs/target/filesystem.md]: Fixed in https://github.com/aquasecurity/trivy/pull/3503/commits/adc9fce4459cc882f49d138b56c5d6a805e45f31

**Review by DmitriyLewen — APPROVED** [review]

**Pull request merged**